packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -96,7 +96,7 @@ class UnthemedGraphNG extends React.Component<GraphNGProps, GraphNGState> {
 
     return {
       ...state,
-      data: preparePlotData(frame, [FieldType.string]),
+      data: preparePlotData(frame, [FieldType.number]),
```

Comment:
if we want a generic uPlot wrapper, then we should probably pass `time` here also. right now `preparePlotData()` assumes we also want `time`, but i'd rather we be explicit about it.

also, maybe `Set` is better than `Array`, since `.has()` is preferable to the current `indexOf()` test used internally.

Reply:
is this needed now?  I would like to see how other bits shake out -- and we only ever actually check a list of 1, so :shrug:

Reply:
we can leave for later